File: Exercises_Python/1064.py
valores = []
for count in range(0,6):
    valores.append(float(input()))

positivos = []
for c in valores:
    if c > 0:
        positivos.append(c)
media = sum(positivos) / len(positivos)

print(len(positivos),'valores positivos')
print('{:.1f}'.format(media))

File: Bee-Crowd/Exercises_C++/1064.cpp
#include <iostream>
using namespace std;

int main() {

    float k;
    double soma;
    int n, count_positivos;
    n = 1;
    soma = 0;
    count_positivos = 0;
    while (n != 7) {
        cin >> k;
        if (k > 0) {
            count_positivos += 1;
            soma += k;
        }
        n += 1;
    }

double media;
media = soma / count_positivos;

printf("%d valores positivos\n", count_positivos);
printf("%.1f\n", media);

}